apps/web/components/examples/blockchain/lesson-progress-card.tsx

```
@@ -0,0 +1,301 @@
+            <span className="text-sm text-gray-500">
+              Complete previous lessons to unlock
+            </span>
+          )}
+          
+          {isUnlocked && !displayCompleted && (
+            <button
+              onClick={handleCompleteLesson}
+              disabled={isLoading}
+              className="px-4 py-2 bg-blue-600 hover:bg-blue-700 disabled:bg-blue-400 text-white text-sm font-medium rounded-md transition-colors"
+            >
+              {isLoading ? 'Completing...' : 'Start Lesson'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)